crates/bevy_ecs/src/system/system.rs

```diff
@@ -59,6 +60,10 @@ pub trait System: Send + Sync + 'static {
     fn default_labels(&self) -> Vec<SystemLabelId> {
         Vec::new()
     }
+    /// Allows users to get the system's last change tick.
+    fn get_last_change_tick(&self) -> u32;
+    /// Allows users to set the system's last change tick.
+    fn set_last_change_tick(&mut self, last_change_tick: u32);
```

Comment:
As mentioned in #5633, I think this deserves some level of warning around the effects this will have on change detection / that setting it manually could break things if you aren't careful.